frontend/src/App.js

```diff
@@ -64,11 +66,10 @@ class App extends Component {
   }
   render() {
     return (
-    //   <div className="App">
-    //     <header className="App-header"></header>
-    //     <RecipeList recipes={recipes} />
-    //   </div>
-    // );
+      <div className="App">
+        <header className="App-header"></header>
+        <RecipeList recipes={recipes} />
```

Comment:
Is this related to the signup page somehow? I think, like the sign in page, we'll want the signup to happen on a separate page with its own route.